package.json

```diff
@@ -482,6 +482,11 @@
           "default":true,
           "description": "Specifies the visibility of the Command Explorer in the PowerShell Side Bar."
         },
+        "powershell.sideBar.CommandExplorerExcludeFilter": {
+          "type":"string",
```

Comment:
I know they said a comma separated list, but we can just use an array, right?

Reply:
IMHO the user experience of a comma separated string is better than for an array. If we put it in as an array, it creates a link to edit in the settings.js. I have been trying to replicate the `file.exclude` that is contributed by vscode itself. I appears to come from this code: https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/files/electron-browser/files.contribution.ts#L192-L216 unfortunately when I replicate it, it still gives an edit in settings.js.